opam

```diff
@@ -14,9 +14,10 @@ build-test: [
   [ "ocaml" "pkg/pkg.ml" "test" ]
 ]
 depends: [
-  "ctypes" { >= "0.11.0" }
+  "ctypes" { >= "0.12.0" }
   "ctypes-foreign" { >= "0.4.0" }
   "hex" { >= "1.0.0" }
+  "integers" { >= "0.2.2" }
```

Comment:
Do we use any feature introduced in `0.2.2`? Otherwise it's not necessary to add this constraint.